types/enzyme/index.d.ts

```diff
@@ -54,22 +54,22 @@ export interface CommonWrapper<P = {}, S = {}> {
     /**
      * Returns whether or not the current wrapper has a node anywhere in it's render tree that looks like the one passed in.
      */
-    contains(node: ReactElement<any> | string): boolean;
+    contains(node: ReactElement<any> | Array<ReactElement<any>> | string): boolean;
 
     /**
      * Returns whether or not a given react element exists in the shallow render tree.
      */
-    containsMatchingElement(node: ReactElement<any>): boolean;
+    containsMatchingElement(node: ReactElement<any> | Array<ReactElement<any>>): boolean;
 
     /**
      * Returns whether or not all the given react elements exists in the shallow render tree
      */
-    containsAllMatchingElements(nodes: Array<ReactElement<any>>): boolean;
+    containsAllMatchingElements(nodes: Array<ReactElement<any>> | Array<Array<ReactElement<any>>>): boolean;
```

Comment:
Should this be a single heterogeneous array type? i.e. `nodes: Array<ReactElement<any> | Array<ReactElement<any>>`
would allow for
```tsx
containsAllMatchingElements([<span />, [<span />, <span />]])
```
Not sure if that's a legit thing that works in enzyme, haven't messed with React 16 much.

Reply:
I'd be ok with that although was following the method signatures inside of the enzyme lib so that it'd be as close as possible to what they are doing there.